src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(cssString) {
+  const rules = cssString.split(';').filter(Boolean);
+  const obj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `obj`. It will decrease code readability. Consider using a more descriptive name for the object that will hold the style properties, such as `styleObject` or `cssProperties`.